app/delete_feature/delete.test.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+var expect = require('chai').expect;
+var request = require('supertest');
+
+var app = require('../../app/');
+var Post = require('../post/Post');
+var database = require('../../lib/mongo/');
+
+
+describe('deleting', function (){
+  var seededPost;
+  beforeEach(function (done){
+    //and one tweet from each
+    var post = {
+      text : 'HelloWorld'
+    };
+    database.connect(function (){
+      Post.collection.insert(post, function (err, data){
+          seededPost = data.ops;
+          done();
+        });
+    });
+  });
+
+  //drop collections after tests
+  after(function (done){
+      Post.dropCollection(done);
+    });
+
+  //  test route
+  it('should respond with success', function (done){
+        request(app)
+        .delete('/post/' + seededPost[0]._id)
+        .expect(200)
+        .end(function (err) {
+          if (err) throw err;
+          done();
+        });
+      });
+
+  //  adding hidden to a post
+  it('should add hidden to the post', function (done){
+      request(app)
+        .delete('/post/' + seededPost[0]._id)
+        .expect(200)
+        .end(function (err){
+          if (err) throw err;
+          Post.findById(seededPost[0]._id, function (err, post){
```

Comment:
For this acceptance test, or one that actually tests HTTP responses, use the output of the html, do not connect to the database. Database calls are used for integration tests.

Preferred: send a delete request, and then make another request to see if the data is visible on the page.